spec/rubocop/cop/style/string_concatenation_spec.rb

```diff
@@ -158,4 +158,26 @@
       RUBY
     end
   end
+
+  context 'empty quotes' do
+    it 'registers offense and corrects' do
+      expect_offense(<<-RUBY)
+        '"' + "foo" + '"'
+        ^^^^^^^^^^^^^^^^^ Prefer string interpolation to string concatenation.
+        '"' + "foo" + "'"
+        ^^^^^^^^^^^^^^^^^ Prefer string interpolation to string concatenation.
+        "'" + "foo" + '"'
+        ^^^^^^^^^^^^^^^^^ Prefer string interpolation to string concatenation.
+        "'" + "foo" + '"' + "bar"
+        ^^^^^^^^^^^^^^^^^^^^^^^^^ Prefer string interpolation to string concatenation.
+      RUBY
+
+      expect_correction(<<-RUBY)
+        "'foo'"
+        "'foo'"
+        "'foo'"
+        "'foo'bar"
+      RUBY
```

Comment:
This doesn't look right to me, it's changing the actual string value now. The original examples returned these values:

```
"foo"
"foo'
'foo"
'foo"bar
```

Reply:
@dvandersluis You are right. I have fixed this now.